.github/workflows/functional-tests-approval.yaml

```diff
@@ -0,0 +1,17 @@
+name: 'Functional test approval using environments'
+on:
+  pull_request:
+    branches:
+      - main
+      - features/*
+      - release/*
+jobs:
+  functional-tests:
```

Comment:
Can you find a name that makes more sense? This isn't actually running the functional tests.